main.cpp

```diff
@@ -474,212 +460,174 @@ static const char *get_conf_dir()
     return DEFAULT_CONF_DIR;
 }
 
-static struct endpoint_config *search_endpoints(const char *name)
+static int parse_mavlink_dialect(const char *val, size_t val_len, void *storage, size_t storage_len)
```

Comment:
World would be a better place if there were a easy way to generate an enum from string...